app/models/infra_conversion_job.rb

```diff
@@ -51,18 +51,10 @@ def migration_task
   end
 
   def start
-    # TransformationCleanup 3 things:
-    #  - kill v2v: ignored because no converion_host is there yet in the original automate-based logic
-    #  - power_on: ignored
-    #  - check_power_on: ignore
-
-    migration_task.preflight_check
-    _log.info(prep_message("Preflight check passed, task.state=#{migration_task.state}. continue ..."))
+    message = "Preflight check is ok. A conversion host has been assigned. Starting."
```

Comment:
If the preflight check is always run first is this log message needed?

Reply:
Not really. Removed.